### PR TITLE
Add hero logo and info

### PR DESCRIPTION
**Pull request by heroworldnft**
https://bscscan.com/tx/0x6fde7cb3484e15e24d4ff5948d8ac845d72195a0a61b365b7fce4d085ccf9f68

Hero is a decentralized digital game token created by combining NFT and Metaverse on the Binance Smart Chain. Anyone can get token rewards through proficient gameplay and contribution to the ecosystem.
A series of functions introduced by the Hero ecosystem include token mortgage, dynamic token rarity, rarity mining, and game mechanisms managed by the decentralized autonomous group (Heroworld).

### CODE REVIEW

File: bsc/0x43605A08cf9987B7fa713879D7873753Ff82D854/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "name":"hero",
+    "website":"https://horeworldnft.com",
+    "description":"Hero is a decentralized digital game token created by combining NFT and Metaverse on the Binance Smart Chain. Anyone can get token rewards through proficient gameplay and contribution to the ecosystem.
+A series of functions introduced by the Hero ecosystem include token mortgage, dynamic token rarity, rarity mining, and game mechanisms managed by the decentralized autonomous group (Heroworld).",
+  "explorer":"https://bscscan.com/token/0x43605A08cf9987B7fa713879D7873753Ff82D854",
+    "type":"BEP20",
+    "symbol":"hero",
+    "decimals":9,
+    "status":"active",
+    "id":"0x43605A08cf9987B7fa713879D7873753Ff82D854"
```
Comment (heroworldnft) [inline bsc/0x43605A08cf9987B7fa713879D7873753Ff82D854/info.json:11]: 交易哈希链接：https://bscscan.com/tx/0x6fde7cb3484e15e24d4ff5948d8ac845d72195a0a61b365b7fce4d085ccf9f68

**Comment by jiasutijiao** [comment]
专业提交logo！审核失败的走加速审核2小时内包过！价格显示问题！QQ:3579750034

**Comment by opera0001** [comment]
1. If the contract is not open source, there may be loopholes
2. Insufficient working capital pool
Please re submit the logo application after adjustment or optimization. Thank you for your support for tokenpocket.

**Comment by opera0001** [comment]
1. Not relinquishing ownership
2. Contract not open source
Please submit again after adjustment. Thank you for your support for tokenpocket

**Pull request closed without merging**